docs/product/performance/mobile-vitals/app-starts.mdx

```diff
@@ -62,6 +62,14 @@ To get additional information about any of your application’s screens, click o
 
 The table below the chart shows spans that have changed from one release to the next and allows you to filter for specific span operations and device classes. Being able to narrow down to specific event samples helps debug slow starts. 
 
+**Eligible span operations:**
```

Comment:
Hi @vivianyentran, thanks for the review and my apologies for getting back to this late.

I get what you mean about how this disrupts the flow of the section. I've gone ahead and moved it to the end of this section but I think it still belongs in this "Screen Summary Page" header because there are more operations that are likely included in the app start time, but we can only collect data for some of those spans (e.g. custom spans can't be indexed because of the way we do metrics right now). With that in mind, this information is specific to that UI element.

I've also changed `Eligible span operations` to `The following table describes the span operations that are surfaced in the spans table` to explain there are specific ops we can view here. Let me know if you think I could be more specific here, I wanted to say we can only show specific ops but didn't want to go into the details about why other ops aren't included.

Reply:
Thanks for clarifying! That makes sense. I think the changes you've made are good to go then!